store/views.py

```diff
@@ -77,6 +124,43 @@ def loanBookView(request):
 @csrf_exempt
 @login_required
 def returnBookView(request):
-    pass
+    if request.method=="POST":
+        try:
+            book_id=request.POST['bid']
+            book=BookCopy.objects.get(pk=book_id)
+            book.status=True
+            book.borrower=None
+            book.borrow_date=None
+            book.save()
```

Comment:
There must be a validation in the backend when a user is returning the book, to make sure that he has only borrowed the book. Otherwise, a simple POST request will make the BookCopy to be returned, and would set its status as True.